README.md

```diff
@@ -108,9 +108,9 @@ fastify.route(route);
 fastify.listen(3000);
 ```
 
-### Notes
+### Route Configuration
 
-The plugin uses the key `routeId` in the `metrics` object of the `config` object in the `Request.context` or `Reply.context`. If none is found a default `noId` string is used.
+The plugin uses the key `routeId` in the `metrics` object of the `config` object in the `Request.context` or `Reply.context`. **WARN**: If none is found the route will not be metricated, and all the metrics methods will be disabled.
```

Comment:
I like the change of the header, I think it's better to change the sentence to say that to configure a route you have to pass a `metrics` object with a `routeId` key with a non-empty string to route configuration object, otherwise the route stats won't be collected and the request reply decorators won't send any metric. We can remove the link to the request/reply context.

Reply:
ok